tests/unit/jobserver/views/test_projects.py

```diff
@@ -199,6 +199,33 @@ def get_repo_is_private(self, *args):
     assert "Public" not in response.rendered_content
 
 
+def test_projectdetail_with_timed_out_github(rf):
+    project = ProjectFactory()
+    WorkspaceFactory(
+        project=project, repo=RepoFactory(url="https://github.com/owner/repo")
+    )
+    WorkspaceFactory(project=project, repo=RepoFactory(url="/path/on/disk/to/repo"))
```

Comment:
Minor nit, but I'm not sure that we need to create two workspaces here. I _think_ we're exercising the same code if we just have one, but I could be wrong.

Reply:
Test seems fine without the second one, I'll remove it then merge.